ROADMAP.md

```diff
@@ -11,28 +11,50 @@ features. Reach us through Issues, Slack and / or Google Group!
+like network flow export are not supported for Windows Node yet. We will
+continue to add more features for Windows, and improve Antrea Agent and OVS
+installation on Windows Nodes.
 
 * **IPv6 Pod network**
 Support IPv6 and IPv4/IPv6 dual-stack for Pod network. Right now Antrea supports
 only IPv4.
 
-* **Network and NetworkPolicy troubleshooting**
-We want to focus on troubleshooting, and will add more mechanisms to simplify
-troubleshooting Antrea network and the NetworkPolicy implementation.
+* **Antrea NetworkPolicy**
+Antrea starts to support [Antrea native NetworkPolicy](/docs/network-policy.md)
+besides K8s NetworkPolicy since version 0.8.0. We already support
+ClusterNetworkPolicy and tiered NetworkPolicy, and will continue to add more
+NetworkPolicy extensions, like traffic logging, policy statistics, policy
+realization status, more matching criteria and actions.
 
-* **Prometheus integration**
-Export metrics to [Prometheus](https://prometheus.io) from both Antrea Controller and Agents.
+* **Network diagnostics and observability**
+Network diagnostics and observability is one area we want to focus on. Antrea
+already implements some useful features on this front, including Octant UI
+plugin, CLI, Traceflow, network flow export and visualization, Prometheus
+metrics, OVS flow dumping and tracing. We will continue to enhance existing
+features and add new features to help diagnosing K8s network and NetworkPolicy
+implementation, and to provide good visibility of Antrea network.
 
-* **Kubernetes Service by OVS**
-Implement Kubernetes ClusterIP Services with OVS. Right now Antrea relies on
-`kube-proxy` to serve ClusterIP Services. The OVS implementation could perform
-better than the `kube-proxy` iptables mode.
+* **Flexible IPAM**
+So far Antrea leverages K8s NodeIPAM for IPAM which allocates a single subnet
+for each K8s Node. In future, Antrea will implement its own IPAM, and support
+more IPAM strategies besides subnet per Node, like IP pool per Node or
+per Namespace.
 
-* **Export OVS flow information**
-Export OVS flow information using protocols like IPFix, NetFlow, sFlow. This is
-for enabling visibility into the Antrea network from a flow analyzer.
+* **Egress policy**
+Egress policy is to control the egress Nodes and SNAT IPs of traffic from Pods
+to external network. For example, user can request a dedicated SNAT IP per
+Namespace, or for a particular set of Pods or Services. This feature is very
+useful for services in the Node or external network to identify the source of
+Pod traffic based on SNAT IP and enforce specific policies on the traffic. Also
+check the [egress policy proposal](https://github.com/vmware-tanzu/antrea/issues/667)
+to learn more.
+
+* **NFV and Telco use cases**
+We plan to explore and provide support for NFV and Telco use cases. Will support
+Multus integration, Pod interfaces on SRIOV devices, and Network Service
+Chaining.
```

Comment:
off topic: the network service chaining: is it something like insert a firewall (or any advanced) service in the path of requests ? because a somewhat similar user story came up in networkpolicy++ meeting, where they want to be able to insert a SecurityGateway for cross namespace communication